ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.4.yaml

```diff
@@ -0,0 +1,410 @@
+base_images:
+  assisted-installer:
+    name: assisted
+    namespace: ocp-kni
+    tag: assisted-installer
+  assisted-installer-agent:
+    name: assisted
+    namespace: ocp-kni
+    tag: assisted-installer-agent
+  assisted-installer-controller:
+    name: assisted
+    namespace: ocp-kni
+    tag: assisted-installer-controller
+  assisted-test-infra:
+    name: assisted-test
+    namespace: ocp-kni
+    tag: assisted-test-infra
+  coreos_centos_stream8:
+    name: stream8
+    namespace: coreos
+    tag: "8"
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openshift_release_golang-1.16:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+  postgresql-12-centos7:
+    name: postgresql-12-centos7
+    namespace: ocp-kni
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: Dockerfile.assisted-service
+  from: coreos_centos_stream8
+  inputs:
+    openshift_release_golang-1.16:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.16
+  to: assisted-service
+- dockerfile_literal: |
+    FROM coreos/centos
+    COPY . .
+  from: src
+  to: assisted-service-scripts
+operator:
+  bundles:
+  - dockerfile_path: deploy/olm-catalog/bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/ocpmetal/assisted-service:latest
```

Comment:
Are we going to change this org in this release timeframe?
Is that okay to do after this is merged?

Reply:
This change would have to happen after merge. We don't have ocm-2.4 images being promoted without the configurations in this PR.

Reply:
To further clarify this will need to be changed to: `quay.io/edge-infrastructure/assisted-service:ocm-2.4` Which currently doesn't exist because we haven't configured the `release-ocm-2.4` branch yet.